content/blog/balance-le-son.md

```diff
@@ -0,0 +1,50 @@
+---
+title: DevFest 2019 - Balance the sound
+brief: Balance the sound
+image: /images/blog/balance-le-son.jpg
+date: 2019-10-07
+draft: false
+---
+
+## Prepare your coming!
+This is the DevFest is coming soon, it's time for you to organize! 🎶 [Burn to run - Bruce Springsteen](https://www.youtube.com/watch?v=IxuThNgl3YA) 🎶
+
+Here is the attendees checklist at DevFest → It's important to have:
+
+* Her **badge always on her** :warning:
+* An ID to withdraw your ticket. His laptop to participate in codelabs (check on the description of the talk installation prerequisites).
+* Its phones and tablets loaded to block for tweeter ([#DevFestNantes](https://twitter.com/search?q=%23devfestnantes&src=typd))!
+* Its **good mood** and remember that there is a [Code Of Conduct](https://devfest.gdgnantes.com/code-of-conduct/) to adhere to for an inclusive and benevolent conference.
+* Well **keep its cup** :warning: (Supplied with the bag given at the entrance). It will serve you coffee, tea and other drinks served during the event.
+* His agenda prepared in advance:
+    * Download the sessions calendar directly on the [website](https://devfest.gdgnantes.com/schedule/),
+    * Or the mobile app: [Google Play](https://play.google.com/store/apps/details?id=io.monkeypatch.mobile.konfetti) - [App Store](https://apps.apple.com/us/app/konfetti/id1480039275?app=itunes)
+    * [Rebroadcasting Area](https://devfest.gdgnantes.com/attending/) expected if the Graslin Room is full
+    * Keep a dedicated time to walk on the stands (exchanges, contests, animations, ...)
+    * Enjoy the Lounge Zone #relaxation
+    * Go listen to a concert on the open stage
+* Logistics with everything you need in the [Practical](https://devfest.gdgnantes.com/attending/) section to get to DevFest or know our partner hotel
+* Avoid the big bags in vigipirate plan period, a search of the bags will be carried out. A cloakroom will be available.
+
+## Removing seats
+There is no need to print your ticket to withdraw your place, but remember to have at least the **digital version of your pass**.
+
+To avoid long queues, you can withdraw your pass **from Saturday, October 19 from 12h to 16h**. This will save you precious time on Monday morning. The doors open at 8am on Monday morning and the keynote will begin at 9am. 🎶 [Human Heat - Christine and the Queens](https://www.youtube.com/watch?v=FkWVD9w98ok) 🎶
```

Comment:
```suggestion
To avoid long queues and save time the Monday morning, join us on  **Saturday, October 19 from 12h to 16h** to register. The doors open at 8am on Monday morning and the keynote will begin at 9am. 🎶 [Human Heat - Christine and the Queens](https://www.youtube.com/watch?v=FkWVD9w98ok) 🎶
```